Update network-watcher-packet-capture-manage-portal.md

ShuheiUda opened the pull request:
When I block inbound traffic from 168.63.129.16, I can't capture packets.
Therefore, NetworkWatcher need to allow inbound traffic from 168.63.129.16.

Error
![image](https://user-images.githubusercontent.com/16984569/35721265-7d861c78-0834-11e8-8b36-6a3eea883245.png)

NSG settings
![image](https://user-images.githubusercontent.com/16984569/35721311-aff0fc8c-0834-11e8-9f0d-a28964cba269.png)

azurecla commented:
Hi __@TwoUnder__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mausher). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

TwoUnder commented:
#sign-off

rmca14 commented:
@barbkess

ShuheiUda committed:
Update network-watcher-packet-capture-manage-portal.md

When I block inbound traffic from 168.63.129.16, I can't capture packets.
Therefore, NetworkWatcher need to allow inbound traffic from 168.63.129.16.

ShuheiUda committed:
Update network-watcher-packet-capture-manage-portal.md
